frontend/src/views/Business.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, Fragment } from 'react';
+import Carousel from '../components/Carousel/Carousel';
+import Button from '../components/Button/Button';
+
+import shareIcon from '../assets/share.svg';
+import backArrowIcon from '../assets/back-arrow.svg';
+import chevronDown from '../assets/chevron-down.svg';
+import businessImage from "../assets/business.jpg";
+import smallBusiness from "../assets/business1.jpg";
+import plusIcon from "../assets/plus.svg";
+
+const businessData = [
+  { name: 'business 1', businessImage },
+  { name: 'business 2', businessImage: smallBusiness },
+  { name: 'business 3', businessImage },
+  { name: 'business 4', businessImage },
+  { name: 'business 5', businessImage },
+]
+
+const products = ['bread', 'teas', 'oils', 'pots'];
+
+const Business = () => {
+  const [showMore, requestMore] = useState(false);
+  const [currentProduct, setCurrentProduct] = useState('bread');
+
+  return (
+  <div className="Business">
+    <header className="Business-header">
+      <div className="Business-header-item">
+        <img src={backArrowIcon} alt="back icon" className="small-icon" /> <span className="pl-1 text-capitalize md-size">back</span>
+      </div>
+      <div className="text-capitalize md-size high-emphasis">menfred's bakery</div>
+      <div>
+        <img className="medium-icon" alt="next icon" src={shareIcon} />
+      </div>
+    </header>
+
+    <section className="Business-showcase">
+      <Carousel carouselData={businessData} />
```

Comment:
Sorry! :( I think you misunderstood the design. There's no need for a carousel. This is the detail view of one business. The others are shown on the map which Kevin develops. Please remove code related to the carousel, unless you see the need for it somewhere else.